packages/spruce-cli/src/stores/SchemaStore.ts

```diff
@@ -47,12 +47,32 @@ export default class SchemaStore extends AbstractStore {
 		]
 
 		// Each skill's slug will be the namespace
-		const templateItems = this.utilities.schema.generateTemplateItems({
+		const coreTemplateItems = this.utilities.schema.generateTemplateItems({
 			namespace: 'core',
 			definitions: schemas
 		})
 
-		return templateItems
+		// Local
+		const localDefinitions = await Promise.all(
+			(
+				await globby([
+					path.join(this.cwd, '/build/src/schemas/*.definition.js'),
```

Comment:
This looks like it will load the same thing - a built and a source file. Can we standardize on only loading from the `src/` directory?

Reply:
Crap, I don't know actually. lets afterparty.